### DISCUSSION TITLE
Airpods Pro and Computer application Recording

### DISCUSSION THREAD

**Discussion by fmischel** [ggml-org/whisper.cpp#1847] [Q&A]
Hi, I normally take calls from zoom or discord on my computer and I use my airpods to use as speakers/mic. Is there a way to record both the output of the application and the input from my airpod microphone?

**Comment by fbv000** [comment]
Any luck with that? I get two files that I need to combine to use it on whisper

**Comment by itsthisjustin** [comment]
You can do this on your own with Soundflower or similar software like this https://rogueamoeba.com/loopback/ (which is what I use) installed. It tricks your mac into thinking your system audio is a mic input. For WhisperC++ to do this in repo doesn't make a lot of sense as it's out of scope for this project.
